tiledb/sm/c_api/tiledb.cc

```diff
@@ -3659,12 +3659,15 @@ int32_t tiledb_deserialize_array_schema_evolution(
     return TILEDB_OOM;
   }
 
+  auto memory_tracker = ctx->context().resources().create_memory_tracker();
+  memory_tracker->set_type(sm::MemoryTrackerType::ARRAY_EVOLVE);
```

Comment:
More crossing of PRs?

Reply:
No, I need to provide this infrastructure for enumerations. However, you're correct and there is overlap.